lib/ferrum/page.rb

```diff
@@ -355,13 +356,21 @@ def subscribe
     end
 
     def prepare_page
```

Comment:
Should I keep everything in this method or split out the worker preparation into a different method?